src/modules/classAwesome.js

```diff
@@ -0,0 +1,63 @@
+export default class Awesome {
+  constructor() {
+    this.data = {};
+    this.record = [];
+  }
+
+  addRecord(chore) {
+    this.data = { chore };
+    this.record.push(this.data);
+    return this.data;
+  }
+
+  // removeRecord(rec) {
+  //   const result = this.record.filter(el => el === rec);
+  // }
```

Comment:
- [ ] Hi, you could remove this commented code to make your codebase slightly cleaner.